www/shared/styles/imports/custom-medias.css

```diff
@@ -0,0 +1,44 @@
+/* ==========================================================================
+   Custom media queries
```

Comment:
Can you add a note:
`> for more informations check https://github.com/postcss/postcss-custom-media` ?

Reply:
I will put this on PR !47